docs/concepts/01_terminologies.md

```diff
@@ -29,6 +29,12 @@ In the [PresentProof](./00_what_is_hl_aries.md#8-presentproof-protocol) flow, th
 ### Issuer
 The entity that issues a [credential](#verifiable-credential) to a holder.
 
+## VDRI
+
+An interface for verifying data against an underlying ledger. A role a system might perform by mediating the creation and verification of relevant data which might be required to use [verifiable credentials](#verifiable-credential).
```

Comment:
It's not always ledger. eg. [DIDKey](https://w3c-ccg.github.io/did-method-key/) is constructed on the fly and [DID Peer](https://identity.foundation/peer-did-method-spec/) is stored in non-ledger backend.

Reply:
true..how's this:

`An interface for verifying data against a trusted datastore such as an underlying ledger, decentralized database or a government ID database.`